torch/csrc/jit/frontend/source_range.h

```diff
@@ -287,6 +280,18 @@ struct TORCH_API Source {
     }
   }
 
+  static StringCordView create_text_view(
+      CopiesString copies_str,
+      std::string_view text_view) {
+    if (copies_str == COPIES_STRING) {
+      auto allocated_str =
+          std::make_shared<std::string>(text_view.data(), text_view.size());
+      return StringCordView({*allocated_str}, {allocated_str});
```

Comment:
Allocated string shouldn't be moved in the second arg?

Reply:
whoops, just saw this. PR is already landed internally and there appears to be some propagation delay, so I can't fix it on this PR.

just adding a `std::move` wouldn't work because order of subexpression evaluation is unspecified, so `*allocated_str` might break. you can still avoid the refcount bump, but I'm probably not going to get to it unless a profiler tells me to because this path is also doing three heap allocations (the StringCordView arguments are vectors) so it's a bit of a lost cause for efficiency anyway. (the TorchLibraryInit path isn't COPIES_STRING)